GenderPayGap.WebUI/Views/Submit/RelatedContent.cshtml

```diff
@@ -1,23 +1,24 @@
 ﻿@using GenderPayGap.Core
 @using GenderPayGap.WebUI.Controllers.Guidance
-<div class="column-third">
-    <aside class="govuk-related-items" role="complementary">
-        <h3 class="heading-small" id="subsection-title">Related content</h3>
+
+<div class="govuk-grid-column-one-third">
+    <aside style="border-top: 2px solid #005ea5; padding-top: 5px;" role="complementary">
+        <h3 class="govuk-heading-s">Related content</h3>
         <nav role="navigation" aria-labelledby="subsection-title">
-            <ul class="font-xsmall">
+            <ul class="govuk-body govuk-body-s govuk-list govuk-list--spaced">
                 <li>
-                    <a rel="external noopener track" target="_blank" data-track-category="Related Link Clicked" href="@Global.GpgGuidanceLink">Gender pay gap reporting<span class="visually-hidden">(opens in a new window)</span></a>
```

Comment:
we should call `sendGpgEvent` declared in `_Tracking.cshtml`

it will be something like this:
`onclick="sendGpgEvent({eventCategory: 'Related Link Clicked', eventAction: '@Url.Action("Overview", "ActionHub")', eventLabel: 'Actions to close the gap'})"`

The category is the same for all the events.
eventAction: the URL
eventLabel: the description